backend/inspirehep/matcher/tasks.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2020 CERN.
+#
+# inspirehep is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+
+from celery import shared_task
+from invenio_db import db
+from invenio_records.api import RecordMetadata
+from sqlalchemy import type_coerce
+from sqlalchemy.dialects.postgresql import JSONB
+
+from inspirehep.records.api import LiteratureRecord
+
+from .api import match_references
+
+
+@shared_task(
+    ignore_result=False, queue="matcher",
+)
```

Comment:
you should add `acks_late=True` and retry options on possible errors (look at  migrator).

Reply:
I guess it will be only DB related errors.